src/sentry/tasks/commit_context.py

```diff
@@ -33,23 +34,31 @@
 logger = logging.getLogger(__name__)
 
 
-def queue_comment_task_if_needed(commit: Commit, group_owner: GroupOwner):
+def queue_comment_task_if_needed(
+    commit: Commit, group_owner: GroupOwner, repo: Repository, installation: IntegrationInstallation
+):
     from sentry.tasks.integrations.github.pr_comment import comment_workflow
 
     logger.info(
         "github.pr_comment.queue_comment_check",
         extra={"organization_id": commit.organization_id, "merge_commit_sha": commit.key},
     )
 
+    response = installation.get_client().get_pullrequest_from_commit(repo=repo.name, sha=commit.key)
```

Comment:
TODO: make sure the client exists and write a test

Reply:
actually, i think it's okay to error if the client doesn't exist OR the API request fails